src/themes/material/theme.module.ts

```diff
@@ -52,11 +46,16 @@ import { PipeModule } from 'ng2-qgrid/pipes';
 		MatCardModule,
 		PipeModule
 	],
-	providers: [
-		ThemeService
-	]
+	providers: [ThemeService],
+	entryComponents: [ThemeComponent]
 })
 export class ThemeModule {
-	constructor() {
+	constructor(
+		theme: ThemeService,
+		componentResolver: ComponentFactoryResolver
+	) {
+		theme.name = 'material';
```

Comment:
Probably, it's a responsibility of client code, because if there are multiple themes required, the last one is used.